src/styles/header.scss

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
```

Comment:
The 'src' property in the '@font-face' rule should point to a valid font file URL, such as a .woff or .ttf file, not a Google Fonts page URL. Please update the URL to point to a valid font file.